flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNodeBase.java

```diff
@@ -20,6 +20,7 @@
 
 import org.apache.flink.api.dag.Transformation;
 import org.apache.flink.table.delegation.Planner;
+import org.apache.flink.table.planner.plan.nodes.exec.common.CommonExecExchange;
```

Comment:
Can we remove the template paramter: P extends Planner, just use Planner instead. 
because currently there is no need on special function on StreamPlanner and BatchPlanner, and in the future, I think we may want StreamExecNode and BatchExecNode can be unified and run in mixed mode.

Reply:
In ExecNode, We have removed type parameter `P`, the new method is `Transformation<T> translateToPlan(Planner planner)`. If we support mixed mode, the BatchPlanner and StreamPlanner also need to refactor. We can refactor this after we have a clearer thought.